widget/select_entry.go

```diff
@@ -56,16 +56,21 @@ func (e *SelectEntry) SetOptions(options []string) {
 		items = append(items, fyne.NewMenuItem(option, func() { e.SetText(option) }))
 	}
 	e.dropDown = fyne.NewMenu("", items...)
-	dropDownButton := NewButton("", func() {
```

Comment:
cleaner is 
```go
var dropDownButton *widget.Button
dropDownButton = NewButton("", func() {
...
```
so you don' t have to create an empty func then overwrite it

Reply:
Ok, but it is not a big difference.